lib/marloss/store.rb

```diff
@@ -110,7 +110,8 @@ def delete_lock(name)
       Marloss.logger.info("Lock for #{name} deleted successfully")
     end
 
-    private def process_id
+    private
```

Comment:
why? I personally prefer the `private` beside the `def` making it easier to read git diffs.

Reply:
In my experience calling private with arguments is uncommon. I'm not sure I have ever seen it done that way actually, this is just me reviewing the code as I went through it.

Reply:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/AccessModifierDeclarations I do prefer `inline` over `group`. I moved to the inline a while ago, a prefer it as is more explicit and it helps with diffs. We use it at work as well.
I actually should add rubocop sometimes this week